gait_mapper/vae.py

```diff
@@ -0,0 +1,70 @@
+import tensorflow as tf
+from gait_mapper import utils
+
+
+class VAE(tf.keras.Model):
+    """Variational autoencoder of the gait mapper."""
+
+    def __init__(self, window=200, dof=6, latent_features=6, alpha=0.1):
```

Comment:
Docstrings are needed for these parameters

Reply:
Good point! Corrected!